### ISSUE TITLE
muffin: Stuttering in clutter thumbnail previews

### ISSUE THREAD

**Issue by potocron** [linuxmint/mint21-beta#47]
When using hot corners (present all windows), while playing media - either streaming or local - on that window's thumbnail, the video stutters until it stops (sound continues). If files are being copied, for example, the progress window stops as well. In previous versions of Mint it did not happen.

**Comment by mantuoliveira** [comment]
I can confirm this. I've just tried with and without hardware acceleration and the result is the same.

**Comment by potocron** [comment]
Yeah that's right thank you I forgot to add that. Also both with intel graphics on my laptop, and with nvidia on the desktop

**Comment by clefebvre** [comment]
I see stutter in GWL thumbs also. It seems to be global since the mutter rebase, thumbs don't get updated as fast or as well as before.

**Comment by potocron** [comment]
Thanks Clem, you and the team are doing a great job, and the new versions are getting better and better. Thank you for your work.

**Comment by spamegg1** [comment]
I can also confirm this, I was about to open a new issue. 

When another window is in focus, the thumbnail of a video player window stutters:
[cinnamon-2022-07-25T124340+0300.webm](https://user-images.githubusercontent.com/4255997/180748756-e30305a6-3ac9-485b-841e-edf5e076d6e8.webm)
Strangely, if the video player window itself is in focus, then the thumbnail plays smoothly:
[cinnamon-2022-07-25T124506+0300.webm](https://user-images.githubusercontent.com/4255997/180748862-b70cc5a8-53c0-48d6-8553-8636cede4208.webm)
This also happens if it's a Firefox window playing a Youtube video, etc.

**Comment by kancolle386** [comment]
I have same issue, too.
I attached image file is not happened the issue, because take a screenshot made addresses the issue.

It seems thumbnail covered area is drawing preview normally and not covered area is not so.
- Red line: covered area, drawing normally
- Yellow line: not covered area, so drawing abnormally 

![Screenshot from 2022-07-27 13-23-46](https://user-images.githubusercontent.com/67724396/181163186-9b51599e-5df8-4e1e-ac4d-f021c2bf176d.png)
